numba/stencils/stencil.py

```diff
@@ -160,33 +176,47 @@ def add_indices_to_kernel(self, kernel, index_names, ndim,
             loc = block.loc
             new_body = []
             for stmt in block.body:
-                if (isinstance(stmt, ir.Assign) and
-                    isinstance(stmt.value, ir.Const)):
+                if (
+                    isinstance(stmt, ir.Assign)
+                    and isinstance(stmt.value, ir.Const)
+                ):
                     if config.DEBUG_ARRAY_OPT >= 1:
                         print("remembering in const_dict", stmt.target.name,
                               stmt.value.value)
                     # Remember consts for use later.
                     const_dict[stmt.target.name] = stmt.value.value
-                if ((isinstance(stmt, ir.Assign)
+                if (
+                    (
+                        isinstance(stmt, ir.Assign)
                         and isinstance(stmt.value, ir.Expr)
                         and stmt.value.op in ['setitem', 'static_setitem']
-                        and stmt.value.value.name in kernel.arg_names) or
-                   (isinstance(stmt, ir.SetItem)
-                        and stmt.target.name in kernel.arg_names)):
-                    raise ValueError("Assignments to arrays passed to stencil " \
-                        "kernels is not allowed.")
-                if (isinstance(stmt, ir.Assign)
-                        and isinstance(stmt.value, ir.Expr)
-                        and stmt.value.op in ['getitem', 'static_getitem']
                         and stmt.value.value.name in kernel.arg_names
-                        and stmt.value.value.name not in standard_indexed):
+                    ) or (
+                        isinstance(stmt, ir.SetItem)
+                        and stmt.target.name in kernel.arg_names
+                    )
+                ):
+                    raise ValueError(
+                        "Assignments to arrays passed to stencil kernels is "
+                        "not allowed."
+                    )
+                if (
+                    isinstance(stmt, ir.Assign)
+                    and isinstance(stmt.value, ir.Expr)
+                    and stmt.value.op in ['getitem', 'static_getitem']
+                    and stmt.value.value.name in kernel.arg_names
+                    and stmt.value.value.name not in standard_indexed
+                ):
```

Comment:
I'm not sure about this change. Previously, it was easy to spot that the two branches were checking doing very similar things with the difference being a set/get change.

Reply:
I can see that it's slightly more difficult to compare the first and second `if` statements, but equally I think the old version of the first `if` was extremely difficult to understand - particularly the indentation made it troublesome to wrap my head around when I first tried to wrap my head around it. And there were some `flake8` complaints about indentation of conditionals too.